app/src/main/res/xml/paths.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is this needed ?

Reply:
Yes, I'm using it to specify the path for the file provider.